src/main/java/com/clanki/ui/Ui.java

```diff
@@ -46,4 +46,6 @@ public void printByeMessage() {
     public void printInvalidInput() {
         System.out.println("The input is in an incorrect format.");
     }
+
+    public void printSuccessfulDelete() {System.out.print("Got it. Deleted the flashcard at index "); }
```

Comment:
You might want to format this according to the style guide. Also I think it's better to `printSuccessfulDelete(int index)` and then

```java
System.out.println("Got it. Deleted the flashcard at index " + index);
```

as all print statements are colocated to the same place.